packages/neos-ui/src/Containers/ContentCanvas/InlineUI/NodeToolbar/Buttons/AddNode/index.js

```diff
@@ -37,9 +39,23 @@ export default class AddNode extends PureComponent {
     handleOpenModalBtnClick() {
         const {
             openAddNodeModal,
-            node
+            fusionPath,
+            contextPath
         } = this.props;
+        const closestCollectionElement = dom.closestNode(
```

Comment:
add a TODO here, that we always assume that closestNode is a collection; and that is kinda fragile I think...